integration-tests/cico_run_tests.sh

```diff
@@ -11,6 +11,8 @@ docker run -t \
     -e F8A_JOB_API_URL=${F8A_JOB_API_URL} \
     -e F8A_ANITYA_API_URL=${F8A_ANITYA_API_URL} \
     -e F8A_GREMLIN_URL=${F8A_GREMLIN_URL} \
+    -e F8A_3SCALE_URL=${F8A_3SCALE_URL}
```

Comment:
@GeetikaBatra a backslash is missing here

Reply:
@samuzzal-choudhury, thanks for pointing out.